### PR TITLE
Fsr2

### PR DESCRIPTION
**Pull request by dmitrii-z**
### PURPOSE
FSR2 support
Resolves: https://amdrender.atlassian.net/browse/RPRUS-157, https://amdrender.atlassian.net/browse/RPRHOUD-53

### CODE REVIEW

**Comment by ARozghon** [comment]
Approved by QA

File: pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py [hunk 1]
```diff
@@ -1037,9 +1056,12 @@ def process_setting(setting):
         production_render_setting_categories = [category for category in render_setting_categories if category['name'] != 'ViewportSettings']
         generate_houdini_ds(install_path, 'Global', production_render_setting_categories)        
         viewport_render_setting_categories = [category for category in render_setting_categories \
-            if category['name'] == 'Sampling' or category['name'] == 'AdaptiveSampling' or category['name'] == 'Denoise' or category['name'] == 'ViewportSettings']
+            if category['name'] == 'RenderQuality' or category['name'] == 'Sampling' or category['name'] == 'AdaptiveSampling' or category['name'] == 'Denoise' or category['name'] == 'ViewportSettings']
         for category in viewport_render_setting_categories:
-            del category['houdini']
+            if category['name'] == 'RenderQuality':
+                for setting in category['settings']:
+                    if setting['name'] == 'core:renderQuality':
```
Comment (DagerD) [inline pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py:1063]: can it be simplified like this?
`viewport_render_setting_categories = (category for category in render_setting_categories \`
`        if category['name'] in ('RenderQuality', 'Sampling', 'AdaptiveSampling', 'Denoise', 'ViewportSettings'))`
`for category in (cat for cat in viewport_render_setting_categories if cat['name'] == 'RenderQuality'):`
`    for setting in (set for set in category['settings'] if set['name'] == 'core:renderQuality'):`
`        ...`

**Review by DagerD — APPROVED** [review]
Code looks ok. One note in comments.

**Comment by Vsevolod1983** [comment]
It looks like you should add definition to render settings schema.
I believe you need to add new render settings to this file:  pxr\imaging\rprUsd\schema.usda

Without that Maya cannot recognize type of newly added parameters and makes UI for these parameters with wrong type (like just plain string edit box instead of dropdown type).

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]
It looks like you should add definition to render settings schema.
I believe you need to add new render settings to this file: pxr\imaging\rprUsd\schema.usda

Without that Maya cannot recognize type of newly added parameters and makes UI for these parameters with wrong type (like just plain string edit box instead of dropdown type).

**Review by Vsevolod1983 — APPROVED** [review]

**Pull request merged**